src/data_loader/population/populationGeoDataFrame.py

```diff
@@ -0,0 +1,106 @@
+## This file is for fetching the population data from datapop and outputting af GeoPandas dataframe with population snapped to nodes.
+import logging
+
+import geopandas as gpd
+import networkx as nx
+import numpy as np
+import osmnx as ox
+import rasterio.warp
+from shapely.geometry import Point
+from tqdm import tqdm
+
+tif_path = "../../../data/population/"
+tif_filename = "dnk_ppp_2020_constrained.tif"
+tif_filenamePath = f"{tif_path}{tif_filename}"
```

Comment:
You can use `DATA_DIR` to get the path to the data directory, so a change in project structure won't affect this file. Also has the added benefit of resolving symlinks and normalising, so it doesn't matter which system you're running on

```suggestion
from data_loader import DATA_DIR

POPULATION_DATA_DIR = DATA_DIR / "population"
POPULATION_DATA_FILE = POPULATION_DATA_DIR / "dnk_ppp_2020_constrained.tif"
```